src/NuGet/BuildNuGets.csx

```diff
@@ -143,16 +143,23 @@ string[] TestPackageNames = {
 // or they will not be published anywhere at all
 var PreReleaseOnlyPackages = new HashSet<string>
 {
+    // Contains the compiler build task, but Microsoft.NET.Compilers
```

Comment:
Should such comments go into the package descriptions themselves, so they are visible on nuget?
Also consider updating our documentation page for packages: https://github.com/dotnet/roslyn/wiki/NuGet-packages

Reply:
These are comments why these packages *aren't* being uploaded to NuGet.org, so I don't think the package description is really the right place to put them. There's some disagreement about people where these comments to go, so I'm going to put them here for now and anyone who wants to copy the info to another place can do so. 😄

Reply:
Ok, makes sense.